docs/project/changelog.md

```diff
@@ -40,30 +44,41 @@ myst:
   can opt into worse performance and better error messages.
   {pr}`4027`
 
-- {{ Fix }} Fixed adding getters/setters to a `PyProxy` with
-  `Object.defineProperty` and improved compliance with JavaScript rules around
-  Proxy traps.
-  {pr}`4033`
-
-- {{ Enhancement }} Adds `check_wasm_magic_number` function to validate `.so`
-  files for WebAssembly (WASM) compatibility.
-  {pr}`4018`
-
 - {{ Enhancement }} Add an example for `loadPyodide` and `pyodide.runPython
 {pr}`4012`, {pr}`4011`
```

Comment:
Probably this doesn't need a changelog because it's in the docs, but @rth suggested that the people who wrote these would appreciate a changelog entry... So if he likes we can leave it I suppose.

Reply:
I'll move to misc